offline/ldap-scim-bridge.md

```diff
@@ -45,16 +39,6 @@ config:
     email: "mail"
 ```
 
-When you get the package:
-
-Copy your values and charts folders into the `Wire-Server` directory you're using.
-
-Copy the container image to all of the kubernetes hosts in your cluster.
-
-Pre-seed the docker container for `ldap-scim-bridge` onto all of your kubernetes hosts.
-```
-sudo bash -c "cat ldap-scim-bridge:0.4.tar.bz2 | docker load"
```

Comment:
we are using containerd now. you also might want to show "copy the file" "ssh in" and "load the container" as separate steps.

Reply:
Image is now downloaded already in the artifact and will be loaded into kubenodes as part of seed-offline-containerd.yaml